_data/sites.json

```diff
@@ -6003,6 +6003,16 @@
         ]
     },
 
+    {
+        "name": "RapidAPI",
+        "url": "https://rapidapi.com/",
```

Comment:
What's the link/email to contact support?

Reply:
I added email to support and link to contact page.